Products/ZenUI3/browser/resources/js/zenoss/JobsWidget.js

```diff
@@ -15,7 +15,10 @@ Ext.ns('Zenoss');
 var REMOTE = Zenoss.remote.JobsRouter;
 
 function lengthOrZero(o) {
-    return o===null ? 0 : o;
+    if (o) {
```

Comment:
Unrelated error that was showing "undefined" jobs remaining under some circumstances